descript/matsiro.md

```diff
@@ -4004,54 +3997,45 @@ $$
+
+All the prognostic and diagnostic variables are calculated in each tile, and the fluxes at the land surface $F$ are averaged:
 $$
 F=F_{lake}f_{lake}+\sum_{i=1}^nF_if_i(1-f_{lake})
 $$
 
 $$
 \sum_{i=1}^nf_i=1
 $$
-where n is 2, $F_{lake}$, $F_1$ and $F_2$ denote fluxes at the land surface of lake, potential vegetation and cropland, $f_{lake}$, $f_1$ and $f_2$ denote their corresponding fractional weights, respectively.
-
-By default, tile scheme is applied in land surface type, but it can be used for multiple purposes.
-
-## Lake
-
-The surface heat and water fluxes over lakes have been calculated as one of the tiles in a grid. The water temperature and mass are predicted for the surface layer (minimum thickness of 1 m) and four subsurface layers, based on the thermal diffusion and mass conversion, considering vertical overturning, evaporation, precipitation, and in-flow from and outflow to rivers.
-
-## Potential Vegetation and Cropland
-
-Both potential vegetation and cropland tiles consist of six soil layers, up to three snow layers, and a single canopy layer, driving predictions of the temperature and amount of water in the canopy, soil, and snow.
-
-Potential vegetation is defined according to the vegetation types of the Simple Biosphere Model 2 (SiB2; Sellers et al. 1996) scheme and has 10 categories including land ice. There is no wetland category for land cover in the original SiB2 vegetation types or soil types.
 
-## Appendix
-
-### Output variables
-
-| Variable     | Description                                                  | Code   | Dimension     | Units |
-| ------------ | ------------------------------------------------------------ | ------ | ------------- | ----- |
-| $F$          | Fluxes at the land surface                                   | MATFLX | -             | -     |
-| $F_{lake}$   | Fluxes at the land surface of lake                           | -      | IJLDIM        | -     |
-| $F_i(i=1,2)$ | Fluxes at the land surface of potential vegetation and cropland | -      | IJLDIM,MULTCY | -     |
-
-### Parameters
-
-
-| PARAMETER    | Description                                            | Code    | Dimension     | Initial values | Units |
-| ------------ | ------------------------------------------------------ | ------- | ------------- | -------------- | ----- |
-| $f_{lake}$   | Fractional weight of lake                              | LKFRAC  | IJLDIM        | -              | -     |
-| $f_i(i=1,2)$ | Fractional weight of potential vegetation and cropland | SFFRAC1 | IJLDIM,MULTCY | -              | -     |
+where n is 2, $F_{lake}$, $F_1$ and $F_2$ denote fluxes at the land surface of lake, potential vegetation and cropland, $f_{lake}$, $f_1$ and $f_2$ denote their corresponding fractional weights, respectively. The sum of fractions from 3 types of tile always equals 1.
```

Comment:
Actually, the sum of fractions from 3 types of tiles (=f(lake) + f(1) + f(2)) does not equals 1.